db/schema.rb

```diff
@@ -11,15 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160127035545) do
+ActiveRecord::Schema.define(version: 20160127193752) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "games", force: :cascade do |t|
```

Comment:
All this game stuff concerns me since the migrations are not at all in the branch. This is probably an artifact of you creating two branches at once. Git starts getting tricky when you try to do this.